client/client.go

```diff
@@ -220,6 +222,89 @@ func (c *Client) SendRequest(method string, rawURL string, data url.Values,
 	return c.doWithErr(req)
 }
 
```

Comment:
OK - I still think we should modify `SendRequest` to have no method body and just instead call 

```
return SendRequestWithContext(context.Background(), method, rawURL, data, headers, body)
```